include/Core/Config.php

```diff
@@ -132,15 +132,33 @@ public static function set($family,$key,$value) {
 		$dbvalue = (is_array($value)?serialize($value):$value);
 		$dbvalue = (is_bool($dbvalue) ? intval($dbvalue) : $dbvalue);
 
-		$ret = q("INSERT INTO `config` ( `cat`, `k`, `v` ) VALUES ( '%s', '%s', '%s' )
-ON DUPLICATE KEY UPDATE `v` = '%s'",
+		// The "INSERT" command is very cost intense. It saves performance to do it this way.
```

Comment:
The "set" is used (for example) when external scripts are started: https://github.com/friendica/friendica/blob/develop/boot.php#L1337 (which is something that is done really often).

There is no danger of double inserts because the "ON DUPLICATE KEY UPDATE" is still there.

If I have several options how to do stuff, I always prefer the fastest one if the fast option is safe and the readability and complexity hadn't suffered too much.